modules/prometheus-source/pkg/prom/config.go

```diff
@@ -88,6 +91,27 @@ func NewOpenCostPrometheusConfigFromEnv() (*OpenCostPrometheusConfig, error) {
 		if err != nil {
 			log.Errorf("%s was set to true but failed to load service-ca.crt: %s", env.KubeRbacProxyEnabledEnvVar, err)
 		}
+	} else if env.IsPromMtlsAuthEnabled() {
+		tlsCaCert = x509.NewCertPool()
+		// The /etc/ssl/cert.pem location is correct for Alpine Linux, the container base used here
+		systemCa, err := os.ReadFile("/etc/ssl/cert.pem")
+		if err != nil {
+			log.Errorf("mTLS options were set but failed to load system CAs: %s", err)
+		} else {
+			tlsCaCert.AppendCertsFromPEM(systemCa)
+		}
+		mTlsCa, err := os.ReadFile(env.GetPromMtlsAuthCAFile())
+		if err != nil {
+			log.Errorf("mTLS options were set but failed to load DB_MTLS_AUTH_CA_FILE: %s", err)
+		} else {
+			tlsCaCert.AppendCertsFromPEM(mTlsCa)
+		}
+		mTlsKeyPair, err := tls.LoadX509KeyPair(env.GetPromMtlsAuthCrtFile(), env.GetPromMtlsAuthKeyFile())
+		if err != nil {
+			log.Errorf("mTLS options were set but failed to load DB_MTLS_AUTH_CRT_FILE or DB_MTLS_AUTH_KEY_FILE: %s", err)
```

Comment:
```suggestion
			log.Errorf("mTLS options were set but failed to load PROM_MTLS_AUTH_CRT_FILE or PROM_MTLS_AUTH_KEY_FILE: %s", err)
```

honestly, bonus points if you can use %s here and just use the var name const for these so we don't lose track of the naming